### PR TITLE
Draft of Inside Tracker case study.

### PR DESCRIPTION
**Pull request by ericbenwa**

### CODE REVIEW

**Comment by craigmcginley** [comment]
### <span aria-hidden="true">✅</span> Deploy Preview for *loving-austin-5f2eff* ready!


|  Name | Link |
|---------------------------------|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | cb358eba6c3b441d9dc9f2a7030382bb8bf3889d |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/loving-austin-5f2eff/deploys/6270281f984adb0009f35cc8 |
|<span aria-hidden="true">😎</span> Deploy Preview | https://deploy-preview-395--loving-austin-5f2eff.netlify.app |
|<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
---

_To edit notification comments on pull requests, go to your [Netlify site settings](https://app.netlify.com/sites/loving-austin-5f2eff/settings/deploys#deploy-notifications)._

**Review by jsonin — APPROVED** [review]

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+<Quote quotee="From Bathroom to Healthroom" quoteeSub="">Data let us live aware of our health. For example, when we are messaged that ice cream is high in fat, high in sugar, and high in cow's milk, we process that as "Okay, that's bad, but so what?" As soon as that message becomes quantified, and not by keeping annoying food diaries and looking up the nutrition content of every bite or sip, we get religion. Data take us by the shoulders and shake. We notice that eating this way, and continuing to do so, has taken days, or even a few years, off our precious life expectancy. That is when the entire world changes. But not before.</Quote>
+
+## Solution
+
+#### Straight-forward information design tops
+
+The reason patients use software like Inside Tracker is to get the data they want, in the right way, with as little delay or distraction as possible. The patient's mission is to understand the bio and nutrition data, be the data, and figure out how to make that data not just be, but to make a difference in my health outcomes.
```
Comment (jen-p) [inline src/case-studies/inside-tracker-nutrition-science.md:79]: The last sentence sounds really weird. What does it mean to be the data, and make the data be? Can we shave it down or break it up? Feels like we're trying too hard to be cool and overcomplicating our words. 

There's also a voice change where the patient becomes me.

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+#### Sequencing detailed results, step-by-step, is easier to digest
+
+Bioassay results combined with precision nutrition advice are data rich. Revealing more and more detail as the patient understands more and more about their health and nutrition is a proven design technique. By deferring some advanced features to secondary actions (or screens), makes learning the service easier and makes errors less frequent.
```
Comment (jen-p) [inline src/case-studies/inside-tracker-nutrition-science.md:83]: the 'and more's are not needed and slow down the sentence

Comment (jen-p) [reply src/case-studies/inside-tracker-nutrition-science.md:83]: By ... to makes... is not grammatically correct, we're missing a subject in there somewhere.

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+![Bloodwork Design](/images/case-studies/inside-tracker/insidetracker_bloodwork_design_lg.jpg)
+
+<Divider />
+
+## Results
+
+#### 10 years in the wild
+
+The bioassay results have resonated with InsideTracker patients as an easy-to-understand return-of-information visualization. The designed technique has been live and in-use for a decade!
```
Comment (jen-p) [inline src/case-studies/inside-tracker-nutrition-science.md:93]: above we have return of information without dashes, I don't think we need them here.

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+## Round 2, a Decade Later
+
+#### Good design repeats itself
+
+10 years later and a 10x bigger company, Inside Tracker asked for another Design bump. This time, the business wanted to grow services around health goals and everyday habits.
```
Comment (jen-p) [inline src/case-studies/inside-tracker-nutrition-science.md:99]: Design shouldn't be capitalized?

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+#### Key Question:
+
+How do we assist the patient with habit-forming goals when it comes to food choices, eating, and daily activities?
+
+## Insights and recommendations
+
+#### Blood draws occur yearly. Food choices happen daily.
+
+A nutrition agent, working on your behalf, to organize your food life and better your health, was a new avenue for Inside Tracker. By incorporating feedback from patients, stakeholders, scientists, and engineers, a software model for behavior and precision nutrition was created. Early concepts for the digital manifestation of the activity model were designed (ie how does Inside Tracker's nutrition agent behave on your phone and works with you). These concepts fed product and development teams next iteration of the software.
```
Comment (jen-p) [inline src/case-studies/inside-tracker-nutrition-science.md:109]: no comma needed after behalf.

Comment (jen-p) [reply src/case-studies/inside-tracker-nutrition-science.md:109]: ie.,

Comment (jen-p) [reply src/case-studies/inside-tracker-nutrition-science.md:109]: the next iteration?

**Review by jen-p — CHANGES_REQUESTED** [review]

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+Inside Tracker is a quantified food and nutrition service, creating a hard science-based food engine, which along side your blood work and lifestyle, can synthesize a hyper-personalized food plan.
+
+#### Show the Results, beautifully
+
+Nutrition science is complicated. How foods interact with your chemistry and what food our particular bodies and minds need needs to be shown in straight-forward methods. It shouldn’t take a Masters degree in Nutrition to interpret your results.
+
+<Quote quotee="From Bathroom to Healthroom" quoteeSub="">Data let us live aware of our health. For example, when we are messaged that ice cream is high in fat, high in sugar, and high in cow's milk, we process that as "Okay, that's bad, but so what?" As soon as that message becomes quantified, and not by keeping annoying food diaries and looking up the nutrition content of every bite or sip, we get religion. Data take us by the shoulders and shake. We notice that eating this way, and continuing to do so, has taken days, or even a few years, off our precious life expectancy. That is when the entire world changes. But not before.</Quote>
```
[the remark this answers is not in the document]
Comment (jsonin) [reply src/case-studies/inside-tracker-nutrition-science.md:73]: It's a quote. Let's not change that particular entry.

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+The food industry adds to the chaos by manufacturing calories that appeal to the species with sugary, salty, and fatty combinations. Big Food also obfuscates good nutritional practices by interfering with food pyramid governance, advertising sugary treats to our youngest residents, plying local governments to accept pizza as a vegetable in school lunches, and participating in corporate welfare by receiving subsidies for growing cheap calories such as sugar, corn, and milk (which should not be everyday human feed). Knowing what food is right for your body, at a detailed level, is a maze.
+
+<Divider />
+
+## Insights & Recommendations
+
+#### Nutrition Science is... emerging
+
+Nutrition science rightly advocates for Michael Pollen's trio. And yet, personalized food choices, generated for you, based on chemistry, DNA, activity, biome, and goals are only just arriving based on peer-reviewed research.
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:65]: got it

File: src/case-studies/inside-tracker-nutrition-science.md (hunk 3 above)
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:93]: got it

File: src/case-studies/inside-tracker-nutrition-science.md (hunk 5 above)
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:109]: got it

File: src/case-studies/inside-tracker-nutrition-science.md (hunk 4 above)
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:99]: got it

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+caption: 'Personalized Food based on your DNA, blood, and habits.'
+categories:
+  - 'precision-medicine-and-genomics'
+  - 'patient-engagement'
+upNext:
+  - 'bathroom-to-healthroom'
+  - 'hgraph'
+  - 'open-source-healthcare'
+hidden: false
+metaDescription: 'Personalized Food based on your DNA, blood, and habits.'
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:14]: got it

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+<!-- prettier-ignore-start -->
+How does InsideTracker show your bioassay information and resulting nutrition needs?  
+... so it's not an IEEE paper  
+... so it's beautifully shown (versus told in TLDR)  
+... so it's a service one can grow into, from beginner to expert  
+... so it actually has better health outcomes.
+<!-- prettier-ignore-end -->
+
+How does the InsideTracker food engine help patients pick the right food, at the right time, for my best health?
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:34]: got it

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+<Divider />
+
+## Insights & Recommendations
+
+#### Nutrition Science is... emerging
+
+Nutrition science rightly advocates for Michael Pollen's trio. And yet, personalized food choices, generated for you, based on chemistry, DNA, activity, biome, and goals are only just arriving based on peer-reviewed research.
+
+Inside Tracker is a quantified food and nutrition service, creating a hard science-based food engine, which along side your blood work and lifestyle, can synthesize a hyper-personalized food plan.
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:67]: got it

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+#### Show the Results, beautifully
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:69]: got it

File: src/case-studies/inside-tracker-nutrition-science.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+Nutrition science is complicated. How foods interact with your chemistry and what food our particular bodies and minds need needs to be shown in straight-forward methods. It shouldn’t take a Masters degree in Nutrition to interpret your results.
```
[the remark this answers is not in the document]
Comment (ericbenwa) [reply src/case-studies/inside-tracker-nutrition-science.md:71]: got it

**Pull request merged**